extensions/amp-a4a/0.1/test/test-a4a-integration.js

```diff
@@ -249,7 +249,7 @@ describe('integration test: a4a', () => {
     const forceCollapseStub =
```

Comment:
Do we have a separate test for `xhrMock` rejecting, rather than returning `null`?  If no, please add one.  If yes, please add a check for `forceCollapseStub` not being called there as well.

There is a separate test for the case where the `xhrMock` returns a valid response, but that response has an empty body, right?  Also for a 204?  And both of those check the `forceCollapseStub`?